src/clj/witan/send/model/input.clj

```diff
@@ -43,7 +42,8 @@
   "Takes in a file path and a schema. Creates a dataset with the file
    data after coercing it using the schema."
   [filepath schema]
-  (-> (load-csv filepath)
+  (-> (try (load-csv filepath)
+           (catch Exception e (str filepath " doesn't exist")))
```

Comment:
I don't think we want to catch here and still return a nil as then we'd be passing a nil through to apply-schema-coercion. I think we can just let the exception bubble to the top where we can do something with `ex-data` if we want https://clojuredocs.org/clojure.core/ex-data